### PR TITLE
Feature branch// Tribute_ page_3rd_Project

### PR DESCRIPTION
**Pull request by Marwa-Sadat**
1-  Created a repositry.
2-  Cloned it with my local machine.
3-  Set up the linters.
4-  Added nessecary html and css codes.
5-  And this is my third project called Tribute- Page.
6-  Making brunch.
7-  Creating a Lintrace file, creating a readme file, creating a styling file, creating an HTML file, etc. were necessary things that were told to me in the project.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  max-width: 400px;
+  height: auto;
+  border-radius: 8px;
+}
+
+#img-caption {
+  color: #666;
+  font-size: 14px;
+  margin-top: 10px;
+}
```
Comment (ahzamir) [inline style.css:42]: - Your img element should have a display of block.
- Your #image should have a max-width of 100%.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @Marwa-Sadat 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Your CSS code is missing two requirements  **(Check the inline comments)**
- Copy your HTML file code and paste it to the freecodecamp website Tribute Page project to see which requirements your project is missing solve those missing requirements and mention me here using @ahzamir to re-review your project.
- Add some design to your elements so you can share your project with others.

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
❌ Great Design implemented.
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
❌ You have followed and completed the Survey Form and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: style.css (hunk 1 above)
Comment (Marwa-Sadat) [reply style.css:42]: Hi sir!  thanks for your feedback, I will follow all my mistakes...

Comment (Marwa-Sadat) [reply style.css:42]: Hi sir! I completed my 3rd project requirements but I don't know what is wrong with my survey form  project @ahzamir

**Comment by ahzamir** [comment]
@Marwa-Sadat 
Why did you merge your branch before I approved your pull-request!?

**Comment by Marwa-Sadat** [comment]
oh! so Sorry sir did not notice

**Comment by Marwa-Sadat** [comment]
What should I do now?

**Comment by Marwa-Sadat** [comment]
@ahzamir

**Pull request merged**